### DISCUSSION TITLE
Override a path tag in the swagger aggregator config from Console

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#251] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v10.0.5

### Description

Hi

I would like to change the tag of different endpoint to group them together. 
Example. I have the following endpoints exposing crud collections:

```
/library/books
/library/reviews
``` 

I would like the API Portal to group them under one tag, named `Library`. 
The Console creates automatically a Swagger Aggregator config with two tags:
- `LibraryBooks`
- `LibraryReviews`

Is it possible to override them with a custom one?

### Actual Outcome

One tag groups two endpoints

### Expected Outcome

Two tags from different endpoints

**Comment by fredmaggiowski** [comment, marked as answer]
Hey, this should be possible using the `swagger-aggregator.json` advanced file and configuring the `transformPath` for those oath you need to change.

It should he well documented here: https://docs.mia-platform.eu/docs/runtime_suite/swagger-aggregator/configuration#configuration-file

**Reply by flower-of-the-bridges** [reply]
Hi @fredmaggiowski thanks for the answer! I'll look into that
